src/test/java/io/stargate/sgv2/jsonapi/fixtures/testdata/LogicalExpressionTestData.java

```diff
@@ -166,9 +222,51 @@ public static Object value(DataType type) {
         return "text-value";
       }
       if (type.equals(DataTypes.DURATION)) {
-        // we handle duration as a string until it gets to the codec
-        return "P1H30M";
+        return "P1H30M"; // Handle duration as a string
+      }
+      if (type.equals(DataTypes.INT)) {
+        return 25;
+      }
+      if (type.equals(DataTypes.BIGINT)) {
+        return 1000000L;
+      }
+      if (type.equals(DataTypes.DECIMAL)) {
+        return 19.99;
+      }
+      if (type.equals(DataTypes.DOUBLE)) {
+        return 4.5d;
+      }
+      if (type.equals(DataTypes.FLOAT)) {
+        return 70.5f;
+      }
+      if (type.equals(DataTypes.SMALLINT)) {
+        return 3;
       }
+      if (type.equals(DataTypes.TINYINT)) {
+        return 1;
+      }
+      if (type.equals(DataTypes.VARINT)) {
+        return 123456789;
```

Comment:
Similarly, `VARINT` maps to `BigInteger` (but driver possibly accepts `Integer` like here?)

Reply:
fixed

Reply:
fixed